cluster/gce/configure-vm.sh

```diff
@@ -273,18 +273,33 @@ function create-salt-auth() {
       echo "${KUBELET_TOKEN},kubelet,kubelet" >> "${KNOWN_TOKENS_FILE}";
       echo "${KUBE_PROXY_TOKEN},kube_proxy,kube_proxy" >> "${KNOWN_TOKENS_FILE}")
 
-    mkdir -p /srv/salt-overlay/salt/kubelet
-    kubelet_auth_file="/srv/salt-overlay/salt/kubelet/kubernetes_auth"
-    (umask 077;
-      echo "{\"BearerToken\": \"${KUBELET_TOKEN}\", \"Insecure\": true }" > "${kubelet_auth_file}")
 
-    mkdir -p /srv/salt-overlay/salt/kube-proxy
-    kube_proxy_kubeconfig_file="/srv/salt-overlay/salt/kube-proxy/kubeconfig"
-    # Make a kubeconfig file with the token.
-    # TODO(etune): put apiserver certs into secret too, and reference from authfile,
-    # so that "Insecure" is not needed.
-    (umask 077;
-    cat > "${kube_proxy_kubeconfig_file}" <<EOF
+    # Generate tokens for other "service accounts".  Append to known_tokens.
+    #
+    # NB: If this list ever changes, this script actually has to
+    # change to detect the existence of this file, kill any deleted
+    # old tokens and add any new tokens (to handle the upgrade case).
+    local -r service_accounts=("system:scheduler" "system:controller_manager" "system:logging" "system:monitoring" "system:dns")
+    for account in "${service_accounts[@]}"; do
+      token=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+      echo "${token},${account},${account}" >> "${KNOWN_TOKENS_FILE}"
+    done
+  fi
+}
+
+function create-salt-node-auth() {
+  mkdir -p /srv/salt-overlay/salt/kubelet
```

Comment:
Done. I figured I'd leave it because it seems idempotent, but I guess no reason to do it twice.

Actually, maybe that's a good question: do we actually want to re-do it every time, if it would be idempotent if it's the same, but allow it to be changed if the config does change?

Also, used single `[` and `]` to match what's been done in the master function, but happy to change them all to double if you'd prefer.